### PR TITLE
PhysTrax: v0.01

### PR DESCRIPTION
**Pull request by ekwawu**
Allows user to record heart rate data and download to csv file.

### CODE REVIEW

File: apps/phystrax/metadata.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  "description": "Tracking physiological measurements to support active learning in classrooms",
+  "tags": "health",
+  "interface": "interface.html",
+  "readme": "README.md",
+  "supports": ["BANGLEJS2"],
+  "storage": [
+    {"name":"phystrax.app.js","url":"app.js"},
+    {"name":"phystrax.img","url":"app-icon.js","evaluate":true}
+  ],
+  "data": [{"wildcard":"heart_rate_data.?.csv"}]
```
Comment (bobrippling) [inline apps/phystrax/metadata.json:15]: Looks like a small issue here

> `App phystrax data file wildcard heart_rate_data.?.csv does not include app ID`

The fix being something like this (we'd also need to update `interface.html` too):

```suggestion
  "data": [{"wildcard":"phystrax_hrm.?.csv"}]
```
(or whatever you'd like to call the file)

**Review by bobrippling — COMMENTED** [review]
Great stuff! Just the lint issue in `metadata.json` if you could resolve that

File: apps/phystrax/metadata.json (hunk 1 above)
Comment (ekwawu) [reply apps/phystrax/metadata.json]: Okay, I can do that. Thanks!

Comment (bobrippling) [reply apps/phystrax/metadata.json]: Cool, I've matched the ones in other sources and merged - thanks for the app!

**Pull request merged**